sdk/rust/nym-sdk/examples/sandbox.rs

```diff
@@ -0,0 +1,32 @@
+use futures::StreamExt;
+use nym_network_defaults::setup_env;
+use nym_sdk::mixnet;
+use nym_sdk::mixnet::MixnetMessageSender;
+
+#[tokio::main]
+async fn main() -> anyhow::Result<()> {
+    nym_bin_common::logging::setup_logging();
+    // relative root is `sdk/rust/nym-sdk/`
+    setup_env(Some("../../../envs/sandbox.env"));
```

Comment:
I'd try loading from env, then fallback here

```suggestion
let env_path = std::env::var("NYM_ENV_PATH").unwrap_or_else(|_| "../../../envs/sandbox.env".to_string());
setup_env(Some(&env_path));
```